worker/lease/manifold/manifold.go

```diff
@@ -77,23 +77,23 @@ type manifoldState struct {
 	store  *lease.Store
 }
 
-func (s *manifoldState) start(context dependency.Context) (worker.Worker, error) {
+func (s *manifoldState) start(ctx dependency.Context) (worker.Worker, error) {
```

Comment:
Off-topic, I wish this was named `dependency.Scope`, the clash with `context.Context` is really hard to discern.